src/ansys/motorcad/core/methods/rpc_methods_lab.py

```diff
@@ -113,3 +124,150 @@ def calculate_duty_cycle_lab(self):
         """Run the Lab duty cycle."""
         method = "CalculateDutyCycle_Lab"
         return self.connection.send_and_receive(method)
+
+    def add_internal_custom_loss(self, name, function, type, thermal_node):
+        """Add an internal custom loss.
+
+        Parameters
+        ----------
+        name : str
+            Name of lab internal custom loss
+        function : str
+            Function of lab internal custom loss
+        type : str
+            Type of lab internal custom loss. Options are Electrical or Mechanical
```

Comment:
```suggestion
            Type of lab internal custom loss. Options are ``Electrical`` or ``Mechanical``
```

Makes the option more clear. e.g.
![image](https://github.com/user-attachments/assets/6b2bc73c-5740-4d01-ab1f-b2ec3ce6cd94)